crates/red_knot_python_semantic/src/types.rs

```diff
@@ -605,104 +605,50 @@ impl<'db> Type<'db> {
     ///
     /// [subtype of]: https://typing.readthedocs.io/en/latest/spec/concepts.html#subtype-supertype-and-type-equivalence
     pub(crate) fn is_subtype_of(self, db: &'db dyn Db, target: Type<'db>) -> bool {
+        // Two equivalent types are always equal to each other.
+        //
+        // "Equivalent to" here means that the two types are both fully static
+        // and can be determined to point to exactly the same set of possible runtime objects.
```

Comment:
```suggestion
        // and point to exactly the same set of possible runtime objects.
```

Reply:
I weakly prefer "described by" or "point to" rather than "name", because it feels slightly off to say that any type has a canonical "name". Perhaps you might say that any type can have multiple names, but that also feels confusing.

`type` (== `Instance("type")`) and `type[object]` (== `SubclassOf("object")`) both describe the same set of possible runtime objects, but I wouldn't really describe either as a more valid name than the other.

Reply:
I don't agree that to "name" a set of objects implies "to uniquely name" a set of objects, or even "to preferentially name" a set of objects. A type can have multiple equally-valid names, and it is still perfectly accurate to say that any one of them "names" the type.

So I still (weakly :) ) prefer "name", and my second favorite options are "refers to" to or "describes". I find "points to" a bit awkward, personally. But I think any of them communicate well enough; as the author of the PR, you get to paint the bikeshed :)

Reply:
I pushed a commit switching the verb to "describe", since it sounds like that's something we're both okay with ;)